.gitignore

```diff
@@ -1,99 +0,0 @@
-# Project ignores
```

Comment:
It had been added by one person as part of a PR; it wasn't a broad group one. Could add it back and add more file types if needed.